packages/syntax-icons/package.json

```diff
@@ -387,5 +388,9 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
```

Comment:
@somethiiing super important - these should be `devDependencies`, not dependencies! Otherwise this will be included in the npm install of `@cambly/syntax-icons`